LeaderboardBackend/Services/IAuthService.cs

```diff
@@ -1,8 +1,11 @@
+using System.Security.Claims;
 using LeaderboardBackend.Models.Entities;
 
 namespace LeaderboardBackend.Services;
 
 public interface IAuthService
 {
 	string GenerateJSONWebToken(User user);
+	public string? GetEmailFromClaims(ClaimsPrincipal claims);
+	public Guid? GetUserIdFromClaims(ClaimsPrincipal claims);
```

Comment:
I feel that we should rename this to `GetIdFromClaims`; should be obvious we're getting the user's ID, right?

Reply:
The claims can contain other IDs in the future. For now it's just the UserId, but making this explicit future proofs it.

Reply:
Sure, let's keep it then